src/Controls/src/Core/Label/Label.Windows.cs

```diff
@@ -1,12 +1,44 @@
 #nullable disable
+using System.Collections.Generic;
 using Microsoft.Maui.Controls.Platform;
+using Microsoft.UI.Xaml.Controls;
 
 namespace Microsoft.Maui.Controls
 {
 	public partial class Label
 	{
-		public static void MapDetectReadingOrderFromContent(LabelHandler handler, Label label) => MapDetectReadingOrderFromContent((ILabelHandler)handler, label);
-		public static void MapText(LabelHandler handler, Label label) => MapText((ILabelHandler)handler, label);
+		TextBlock _textBlock;
```

Comment:
Why use this field? The handler is a `LabelHandler` and is always a `TextBlock`. In fact, `ILabelHandler` is a `TextBlock` too.